std/haxe/io/Bytes.hx

```diff
@@ -448,8 +448,13 @@ class Bytes {
 		return s.toString();
 	}
 
-	public inline function getData() : BytesData {
+	#if php @:extern #end public inline function getData() : BytesData {
+		#if php
+		var r = this;
```

Comment:
Sure, I missed that. The `@:extern` is still necessary, though, because `return &$this` is no valid PHP.